app/services/movie_service.rb

```diff
@@ -0,0 +1,41 @@
+class MovieService
+  def self.top_forty
```

Comment:
Some ideas for making this smaller:
1. using a private section with a def self.conn method with our Faraday.get/new inside of it
2. all the params might be storable in other self.methods

Reply:
`
def self.conn
     Faraday.new(url: "") do |faraday|
       faraday.headers["X-API-Key"] = ENV["KEY"]
     end
end
`